test/system/users/edit_user_test.rb

```diff
@@ -27,7 +27,7 @@ def test_update_of_email_requires_confirmation
     fill_in('user[email]', with: 'updated-email@auction.test')
     fill_in('user[current_password]', with: 'password123')
     click_link_or_button('Update')
-    assert(page.has_css?('div.notice', text: 'Updated successfully.'))
+    assert(page.has_css?('div.notice', text: 'Confirmation link was sent to new email address. Please confirm the address for the change to take an effect!'))
```

Comment:
Do we need to rely on HTML structure here? What if we just test it from pure user's perspective?

Like `assert_text 'Confirmation link was sent to new email address. Please confirm the address for the change to take an effect!'`?

Reply:
Great idea, I'll do that.